cedar-policy-core/src/ast/entity.rs

```diff
@@ -354,6 +404,58 @@ impl std::fmt::Display for Entity {
     }
 }
 
+/// `PartialValue`, but serialized as a RestrictedExpr, for historical reasons
```

Comment:
Might add in the comment the fact that we can't serialize all values (specifically extension values) so we have to convert back to exprs

Reply:
Fixed